mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -439,6 +439,13 @@ func resourceContainerCluster() *schema.Resource {
 										Default:     "default",
 										Description: `The Google Cloud Platform Service Account to be used by the node VMs.`,
 									},
+									"image_type": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Default:     "cos_containerd",
+										Description: `The default image type used by NAP once a new node pool is being created.`,
+										ValidateFunc: validation.StringInSlice([]string{"cos_containerd", "cos", "ubuntu_containerd", "ubuntu"}, false),
```

Comment:
Sorry again for my delay on this, tests looked good for the most part! The API must return the value in all caps, so we'll want to set pass `ignoreCase` as `true` here and make the default and validations capitalized.
```suggestion
										ValidateFunc: validation.StringInSlice([]string{"cos_containerd", "cos", "ubuntu_containerd", "ubuntu"}, true),
```

Reply:
@megan07 No worries, I committed the suggestion, thank you!